plugins/source/gcp/codegen/recipes/iam.go

```diff
@@ -9,10 +9,8 @@ import (
 
 var iamResources = []*Resource{
 	{
-		SubService:   "roles",
-		Struct:       &iam.Role{},
-		NewFunction:  iam.NewProjectsRolesService,
-		ListFunction: (&iam.ProjectsRolesService{}).List,
```

Comment:
Is this not used ?

Reply:
> Is this not used ?

@yevgenypats because `SkipFetch` is set to true on all resources below.